code/controllers/subsystems/initialization/computer_networks.dm

```diff
@@ -12,6 +14,8 @@ SUBSYSTEM_DEF(networking)
 	var/list/connection_queue = list()
 	var/list/tmp_queue = list()
 
+	var/datum/extension/network_device/last_failed_device
```

Comment:
Ctrl-F isn't showing if you've added checking/unsetting this in Destroy() for the type but if not please do so.

Reply:
This was an artifact from testing, thanks for catching it. I went ahead and removed it.